tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -0,0 +1,231 @@
+const { test, expect } = require( '@playwright/test' );
```

Comment:
There seems to have been a merging issue by Tziv. I am trying to fix that first.

Reply:
o.k

Reply:
I think that the tests file name should be `nested-tabs.test.js`